Validation/RecoTrack/plugins/TrackingNtuple.cc

```diff
@@ -2657,6 +2672,8 @@ void TrackingNtuple::fillPixelHits(const edm::Event& iEvent,
       pix_zx   .push_back( ttrh->globalPositionError().czx() );
       pix_radL .push_back( ttrh->surface()->mediumProperties().radLen() );
       pix_bbxi .push_back( ttrh->surface()->mediumProperties().xi() );
+      pix_clustSizeCol.push_back( hit->cluster()->sizeY() );
+      pix_clustSizeRow.push_back( hit->cluster()->sizeX() );
```

Comment:
note to self of where the naming is from
https://github.com/slava77/cmssw/blob/c345719fd55032e6894f67a51d8b91374555ad86/DataFormats/SiPixelCluster/interface/SiPixelCluster.h#L124-L128